### PR TITLE
fix: corrects image stretching on Safari browser

### LINKED ISSUE #8713: In 2.0, when using Safari, images are stretched when loaded from a server [need to backport fix to 2.0]
Description update Aug 25:
This was fixed in develop after 2.0 maintenance was created, but wasn't merged to maintenance at the time. It was also not backported, see history below.
This ticket is just about backporting the fix to 2.0.
----
Go to our ccv2 with Safari or ios
https://jsapps.c432wmya2v-teamspart1-s2-public.model-t.myhybris.cloud/electronics-spa/en/USD/
or http://cx-qa16181492.eastus.cloudapp.azure.com:4200/electronics-spa/en/USD/
Some images are stretched
Doesn't happen with Chrome or Firefox
image

CCv2 is using 2.0.... something, from 3-4 weeks ago.
Also happens in latest develop when viewing on a server, but not when Spartacus app is run locally and pointing to public backend server.

Not really related to #8681

### PR DESCRIPTION
**Pull request by morganm58**
closes GH-8713-backport

### CODE REVIEW

File: projects/storefrontstyles/scss/layout/page-templates/_login-page.scss [hunk 1]
```diff
@@ -16,5 +16,9 @@
 
   @include media-breakpoint-down(md) {
     --cx-max-width: 100%;
+
+    button[type='button'] {
+      margin-bottom: 1em;
```
Comment (tobi-or-not-tobi) [inline projects/storefrontstyles/scss/layout/page-templates/_login-page.scss:21]: how is this related?

File: projects/storefrontstyles/scss/layout/page-templates/_landing-page-2.scss [hunk 2]
```diff
@@ -5,6 +5,15 @@
   margin: auto;
 
   cx-page-slot {
+    cx-banner {
+      cx-media {
+        img {
+          display: flex;
```
Comment (tobi-or-not-tobi) [inline projects/storefrontstyles/scss/layout/page-templates/_landing-page-2.scss:11]: I'd rather keep a generic style rule set for all media in Spartacus. Can we move/adapt the rules in `_media.scss` instead?

File: projects/storefrontstyles/scss/components/content/media/_media.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 %cx-media {
   position: relative;
-  display: flex;
-  align-items: stretch;
+  display: block;
```
Comment (tobi-or-not-tobi) [inline projects/storefrontstyles/scss/components/content/media/_media.scss:3]: is this needed? I'd rather not be opinionated about the display mode, can we do that.

**Review by tobi-or-not-tobi — DISMISSED** [review]

File: projects/storefrontstyles/scss/components/content/media/_media.scss (hunk 3 above)
Comment (morganm58) [reply projects/storefrontstyles/scss/components/content/media/_media.scss:3]: All of these changes are in develop back in may. We are trying to get this back ported into 2.0.x to fix the stretchy images. So if there needs to be changes that would have to be done in another ticket

Comment (developpeurweb) [reply projects/storefrontstyles/scss/components/content/media/_media.scss:3]: @tobi-or-not-tobi we're only backporting what was missed from `develop` since May, but I'll take note of your comments for later improvements.

**Review by developpeurweb — APPROVED** [review]
